components/worker/Dockerfile

```
@@ -60,5 +67,9 @@ USER appuser
 # Expose the application port
 EXPOSE 4019
 
+# Health check
+HEALTHCHECK --interval=30s --timeout=5s --start-period=10s --retries=3 \
```

Hadolint (2.15.1)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)